### PR TITLE
data:AgaKadela

### PR DESCRIPTION
**Pull request by agakadela**

### CODE REVIEW

File: data/AgaKadela.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/agakadela",
+      "icon": "FaGithub"
+    },
+    {
+       "name": "Follow me on LinkedIn",
+       "url": "https://www.linkedin.com/in/agakadela/",
+       "icon": "FaLinkedin"
+     },
```
Comment (ChinmayMhatre) [inline data/AgaKadela.json:22]:
```suggestion
     }
```
Please Remove this comma

File: data/AgaKadela.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+   "tags": [
+     "Open Source",
+     "JavaScript",
+     "ReactJS",
+     "NextJS",
+     "Python"
+ ],
```
Comment (ChinmayMhatre) [inline data/AgaKadela.json:30]:
```suggestion
 ]
```
Please remove this comma

File: data/AgaKadela.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Aga Kadela",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "JavaScript Web Developer helping women become ✨ coding superheroes ✨ at Coder Woman",
+  "avatar": "https://avatars.githubusercontent.com/u/14938482?v=4",
```
Comment (ChinmayMhatre) [inline data/AgaKadela.json:6]:
```suggestion
  "avatar": "https://github.com/agakadela.png",
```
Please use this as your avatar

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
I have suggested some changes which could be why the action is failing. You could also rename the json file to `agakadela.json` all in lowercase since you need to match your username. Thank you !

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/agakadela.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/agakadela",
+      "icon": "FaGithub"
+    },
+    {
+       "name": "Follow me on LinkedIn",
+       "url": "https://www.linkedin.com/in/agakadela/",
+       "icon": "FaLinkedin"
+     }
+  ]
```
[the remark this answers is not in the document]
Comment (agakadela) [reply data/agakadela.json]: Thank you, now it works!

File: data/AgaKadela.json (hunk 3 above)
Comment (agakadela) [reply data/AgaKadela.json]: Done

File: data/AgaKadela.json (hunk 2 above)
Comment (agakadela) [reply data/AgaKadela.json]: Done

File: data/AgaKadela.json (hunk 1 above)
Comment (agakadela) [reply data/AgaKadela.json]: Done

**Review by ChinmayMhatre — APPROVED** [review]
Awesome! Looks good.

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍 

Later if you like you can add milestones, events and testimonials - docs can be found here to learn more https://linkfree.eddiehub.io/docs

**Pull request merged**